app/helpers/application_helper.rb

```diff
@@ -16,4 +16,13 @@ def header_image
     # FIXME needs to parse frontpatter
     "media/images/main-header.jpg"
   end
+
+  def hero_image_path(imgpath)
+    if imgpath =~ %r(^/assets/)
+        imgpath
```

Comment:
Guessing the linter may complain about the extra whitespace?

Reply:
Turns out the linter doesn't care :)

Reply:
lol